values.yaml

```
@@ -38,6 +38,15 @@ server:
   replicas: 3
   bootstrapExpect: 3 # Should <= replicas count
 
+  # extraEnvVars is a list of extra enviroment variables to set with the stateful set. These could be
+  # used to include proxy settings required for cloud auto-join feature, 
+  # in case kubernetes cluster is behind egress http proxies. Additionally, it could be used to configure
+  # custom consul parameters.
+  extraEnvironmentVars: {}
+    # http_proxy: http://localhost:3128
```

```suggestion
    # http_proxy: http://localhost:3128,
```

This set of example text is slightly misleading, since commas are required between each key/value pair.